src/zcash/circuit/utils.tcc

```diff
@@ -28,6 +31,14 @@ std::vector<bool> uint256_to_bool_vector(uint256 input) {
     return output_bv;
 }
 
+std::vector<bool> uint256_to_bool_vector(uint256 input) {
+    return to_bool_vector(input);
+}
+
+std::vector<bool> uint252_to_bool_vector(uint252 input) {
+    return trailing252(to_bool_vector(input));
```

Comment:
Instead of using templates we could do `return trailing252(to_bool_vector(input.inner()));` but the way it is now is fine.